src/web/app/pages-instructor/instructor-student-activity-logs/instructor-student-activity-logs.component.html

```diff
@@ -0,0 +1,123 @@
+<h1>Student Activity Logs For {{course.courseName}}</h1>
+<p>This page allows you to find when your students have accessed or submitted a particular feedback session for a given course. Note that you can only view student logs if you have <b>owner or manager privileges</b> for the course. The earliest date you can search for is <b>{{this.LOGS_RETENTION_PERIOD}} days</b> before today.</p>
+<hr/>
+
+<div *tmIsLoading="isLoading" class="card bg-form no-border">
+  <div class="card-body">
+    <div class="row form-group">
+      <div class="col-4">
+        <div class="col-12">
+          <label for="course-id-dropdown" class="font-weight-bold">Activity Type</label>
+        </div>
+        <div class="col-12">
+          <select id="course-id-dropdown" class="form-control col-12" [(ngModel)]="formModel.logType" (ngModelChange)="triggerDefaultLogActivityTypeChange($event)">
+            <option value="" disabled selected hidden>Select Activity Type</option>
+            <option *ngFor="let logType of LOG_TYPES" [value]="logType.value">
+              {{logType.label}}
+            </option>
+          </select>
+        </div>
+      </div>
+      <div class="col-8">
+        <div class="col-12 mb-1">
+          <label for="logs-from-datepicker" class="font-weight-bold">Search period from</label>
+        </div>
+        <div class="input-group">
+          <div id="logs-from-datepicker" class="col-md-6 input-group">
+            <input type="text" class="form-control" ngbDatepicker readonly [minDate]="earliestSearchDate" [maxDate]="formModel.logsDateTo" [(ngModel)]="formModel.logsDateFrom" #logsFromDp="ngbDatepicker"/>
+            <button class="btn btn-light" (click)="logsFromDp.toggle()" type="button">
+              <i class="fas fa-calendar-alt"></i>
+            </button>
+          </div>
+          <div class="col-md-3">
+            <tm-timepicker id="logs-from-timepicker" [(time)]="formModel.logsTimeFrom"></tm-timepicker>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row form-group">
+      <div class="col-4">
+        <div class="col-12">
+          <label for="student-name-dropdown" class="font-weight-bold">Student Name (Optional)</label>
+        </div>
+        <div class="col-12">
+          <select id="student-name-dropdown" class="form-control col-12" [(ngModel)]="formModel.studentEmail">
+            <option *ngFor="let student of students" [value]="student.email">
+              {{student.name}} {{student.email ? '&lt;' + student.email + '&gt;' : ''}}
```

Comment:
Is `lt` and `gt` needed? If they are in quotes they should be fine being just `<` and `>`?